machine.go

```diff
@@ -78,6 +78,9 @@ type Config struct {
 	// should be created.
 	SocketPath string
 
+	// LogPath defines the file path where the Firecracker log is located.
+	LogPath string
+
 	// LogFifo defines the file path where the Firecracker log named-pipe should
 	// be located.
 	LogFifo string
```

Comment:
@xibz @IRCody I'm supporting both LogPath (what Firecracker supports now) and LogFifo (what Firecracker supported before) here. What do you think? Would it be better to let clients break and keep our client thin?

Reply:
I agree lets not break clients for this.